docs/usage.md

```diff
@@ -298,9 +298,9 @@ MALT does not support paired-end reads alignment (unlike other tools), therefore
 
 Krona can only be run on MALT output if path to Krona taxonomy database supplied to `--krona_taxonomy_directory`. Therefore if you do not supply the a Krona directory, Krona plots will not be produced for MALT.
 
-##### MetaPhlAn3
+##### MetaPhlAn
 
-MetaPhlAn3 currently does not accept FASTA files as input, therefore no output will be produced for these input files.
+MetaPhlAn4 is compatible with the MetaPhlAn3 database by adding the `--mpa3` paramter to the MetaPhlAn process in the config file `module.config`.
```

Comment:
The `--mpa3` should go into the `db_params` of the `databases.csv` NOT `ext.args` in modules.config

We do it a bit differently to most nf-core pipelines in that way

Reply:
Yes, right. I should have noticed that!!! Thanks!